src/Hosting/Server.IntegrationTesting/src/Deployers/RemoteWindowsDeployer/RemoteWindowsDeployer.cs

```diff
@@ -123,7 +123,9 @@ public override void Dispose()
             }
             catch (Exception ex)
             {
+#pragma warning disable CA2017 // Parameter count mismatch
                 Logger.LogWarning(0, "Failed to stop the server.", ex);
```

Comment:
Why not fix it instead?
I assume it's supposed to be `LogWarning(0, ex, "...")`

Reply:
Thanks! I will change it to `LogWarning(0, ex, "...")` then.

Reply:
Could you please file an issue for fixing `$""`? You would describe it better than me since you have more context.